### PR TITLE
feat: adds the contributing md for the salty simulations GitHub

### PR DESCRIPTION
**Pull request by owen2007**
adds the contributing MD for the salty simulations discord bot GitHub.

discord:
owen_#4883

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# Contributing
+
+When contributing to this repository please adhere to the guidelines below.
+
+# Mandatory Requirments
+
+- Every line of code should be written profesionally and clearly
+- Before creating a PR proofread your work and make sure it is ready for review
+- Create your commands in the category relative to the command
```
Comment (peregrine-developments) [inline docs/CONTRIBUTING.md]: "category relevant to the command" sounds better here imo

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+// FlyByWire Discord Bot
+// Author: flybywire
+// URL: https://github.com/flybywiresim/discord-bot/
+//
+//============================================================
+//
+//
+//
+//============================================================
+```
```
Comment (peregrine-developments) [inline docs/CONTRIBUTING.md:103]: Mention somewhere around here that the date should be updated to the date of file creation

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- 1. open a command prompt/terminal in your repo directory
+- 2. Run `npm run dev`
+- 3. If all has gone well, you will see the Bot is running as 'http://localhost:3000' and logged into the name of the Bot you created!
+- 4. You can now test your commands
+
+# Editing Methods
+
+### Adding a new command
+
+- 1. Create a new file in the relevant folder `src/commands/` and name it appropriately. `yourcommand.ts`
```
Comment (peregrine-developments) [inline docs/CONTRIBUTING.md:80]: Some clarification about adding new categories/to existing categories would be nice here

- Adding new category: create a new directory in /src/commands/
- Adding to existing category: add command files to an existing category directory

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+- 4. Develop or edit the command that you are working on
+- 5. Test your build locally (Instructions below)
+- 6. Create a PR to merge your command into staging (Template at bottom)
+
+# Testing your build
+
+### node and npm install
+
+- 1. [install node](https://nodejs.org/en/download/), npm is bundled with the download.
+- 2. Open a command prompt or terminal in your repo directory and type `npm install`.
```
Comment (peregrine-developments) [inline docs/CONTRIBUTING.md:40]: Development of the bot also requires node-ts, as the source is typescript

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+- Refrain from any use of exclamation points (!) unless it is for a specific reason e.g. (warnings)
+- Please use the proper capitilisation and names for 3rd party services e.g. (A32NX and not a32nx)
+- Ensure that all spelling,punctuation and grammar are correct
+- Ask a Bot-Developer for help if you are unsure or stuck with anything when writing your command
+- Send a message in #Discord-Bot when beginning work on a command or if you are interested in adding a command that has not yet been identified
+
+# Creating a Pull request (PR)
+
+Reminder: all PRs are pushed into 'staging' before being merged into the 'master' branch
```
Comment (peregrine-developments) [inline docs/CONTRIBUTING.md:26]: _There's a little bit of irony here too_

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+# Writing Guidelines
+
+All guidelines created for this repository must be followed at all times to ensure the smooth operation of the bot not only for the developers but also for the community.
+
+Please refer to the listings below to achieve this.
+
+- Refrain from any use of exclamation points (!) unless it is for a specific reason e.g. (warnings)
+- Please use the proper capitilisation and names for 3rd party services e.g. (A32NX and not a32nx)
+- Ensure that all spelling,punctuation and grammar are correct
```
Comment (peregrine-developments) [inline docs/CONTRIBUTING.md]: There's a bit of irony here because there's a space missing

File: docs/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+- Test your build locally before submitting anything for review
+
+# Writing Guidelines
+
+All guidelines created for this repository must be followed at all times to ensure the smooth operation of the bot not only for the developers but also for the community.
+
+Please refer to the listings below to achieve this.
+
+- Refrain from any use of exclamation points (!) unless it is for a specific reason e.g. (warnings)
+- Please use the proper capitilisation and names for 3rd party services e.g. (A32NX and not a32nx)
```
Comment (peregrine-developments) [inline docs/CONTRIBUTING.md]: A32NX 🤔

**Review by peregrine-developments — CHANGES_REQUESTED** [review]

File: docs/CONTRIBUTING.md (hunk 7 above)
Comment (owen2007) [reply docs/CONTRIBUTING.md]: Ack I forgor what repo this was :/

File: docs/CONTRIBUTING.md (hunk 4 above)
Comment (owen2007) [reply docs/CONTRIBUTING.md:40]: isn't that the same as the FBW bot? but we just don't do it.

**Pull request closed without merging**